src/shared/components/plots/BoxScatterPlot.tsx

```diff
@@ -1014,3 +1014,72 @@ export function toBoxPlotData<D extends IBaseBoxScatterPlotPoint>(
             );
         });
 }
+
+export function toDataDescriptive<D extends IBaseBoxScatterPlotPoint>(
+    data: IBoxScatterPlotData<D>[],
+    logScale?: any
```

Comment:
Same here, could we try to create a specific data structure instead of using `any`? It would be much harder to maintain certain code if we have too many uncertain types inside our codebase